test/plugin/cesium/sync/dynamicmultipolygonconverter.test.js

```diff
@@ -0,0 +1,87 @@
+goog.require('ol.geom.MultiPolygon');
+goog.require('ol.proj');
+goog.require('ol.style.Style');
+goog.require('os.feature.DynamicFeature');
+goog.require('os.layer.Vector');
+goog.require('os.map');
+goog.require('os.proj');
+goog.require('plugin.cesium.VectorContext');
+goog.require('plugin.cesium.sync.DynamicMultiPolygonConverter');
+goog.require('test.plugin.cesium.scene');
+
+describe('plugin.cesium.sync.DynamicMultiPolygonConverter', () => {
+  const {getFakeScene} = goog.module.get('test.plugin.cesium.scene');
+  const VectorContext = goog.module.get('plugin.cesium.VectorContext');
+  const DynamicMultiPolygonConverter = goog.module.get('plugin.cesium.sync.DynamicMultiPolygonConverter');
+  const converter = new DynamicMultiPolygonConverter();
+
+  let feature;
+  let geometry;
+  let style;
+  let context;
+
+  beforeEach(() => {
+    const coords = [
+      [
+        [[0, 0], [0, 5], [5, 5], [5, 0], [0, 0]]
+      ],
+      [
+        [[10, 10], [10, 20], [20, 20], [20, 10], [10, 10]]
+      ]
+    ];
+    geometry = new ol.geom.MultiPolygon(coords);
+    feature = new os.feature.DynamicFeature(geometry);
+    style = new ol.style.Style();
+    layer = new os.layer.Vector();
```

Comment:
Nit: A couple more require()'s os.layer, os.feature

Reply:
These are all required correctly. The base OL classes and `DynamicFeature` are not modules yet and so I don't think they can be included with `goog.module.get`.